gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/main/java/io/gravitee/apim/core/api/model/mapper/ApiMigration.java

```diff
@@ -178,6 +179,14 @@ private List<Property> mapProperties(@Nullable Properties properties) {
             : stream(properties.getProperties()).map(a -> new Property(a.getKey(), a.getValue(), a.isEncrypted(), a.isDynamic())).toList();
     }
 
+    private List<Resource> mapResources(List<io.gravitee.definition.model.plugins.resources.Resource> resources) {
+        return java.util.Optional.ofNullable(resources).stream().flatMap(List::stream).map(this::toV4Resource).toList();
```

Comment:
```suggestion
        return stream(resources).map(this::toV4Resource).toList();
```

we have a mapper to simplify it